### ISSUE TITLE
Github action fails but it succeeds locally

### ISSUE THREAD

**Issue by max-rocket-internet** [avto-dev/markdown-lint#13]
## Describe the bug

Our github actions started failing even though we pin to version `v1.3.1`. An example is this PR: https://github.com/deliveryhero/helm-charts/pull/120

But locally when I run the tool it doesn't complain:

```
$ docker run --rm -v "$PWD:/helm-charts" avtodev/markdown-lint:v1.3.1 --config /helm-charts/ci/markdown-lint.yaml /helm-charts/**/*.md
$ echo $?
0
```

Our config is here: https://github.com/deliveryhero/helm-charts/blob/master/.github/workflows/markdown-lint.yml
And here: https://github.com/deliveryhero/helm-charts/blob/master/ci/markdown-lint.yaml

If you look at the output [here](https://github.com/deliveryhero/helm-charts/runs/2014549844?check_suite_focus=true) you can see it's like it's ignoring our config file.

This is new behaviour. Did something change recently?

### Expected behaviour

Exit with no error and action is successful.

### Actual behaviour

Actions are failing.

### Steps to reproduce the behavior

Look at our PRs with failing actions but no markdown errors

**Comment by tarampampam** [comment]
Hi @max-rocket-internet!

- Try to use latest version (`1.5.0` at this moment)
- Try to run action with env `DEBUG=true` and show the output

**Comment by max-rocket-internet** [comment]
Now it started working again with no changes. Very strange, I guess some github issue after the Actions outage they had. Thanks for the quick response @tarampampam, we get a lot of use out of your tool 😃
